packages/sanity/src/core/form/inputs/arrays/ArrayOfPrimitivesInput/ArrayOfPrimitivesInput.tsx

```diff
@@ -171,11 +171,11 @@ export class ArrayOfPrimitivesInput extends PureComponent<ArrayOfPrimitivesInput
     return (
       <Stack space={2} data-testid="array-primitives-input">
         <UploadTargetCard
-          types={schemaType.of}
-          resolveUploader={resolveUploader}
-          onUpload={onUpload}
           {...elementProps}
+          onUpload={onUpload}
+          resolveUploader={resolveUploader}
           tabIndex={0}
```

Comment:
was it intentional to always override elementProps here? Not sure if it matters, but earlier elementProps could include types, resolveUploader and onUpload and that way override defaults, so this could potentially be either breaking or fixing a bug or both.